MAINTAINERS.MD

```diff
@@ -6,3 +6,5 @@
 | China Mobile | Hanbei Wang |
 | Huawei | Kai Zhang |
 | Huawei | Shuting Qing |
+| Ericsson | Thorsten Lohmar |
+| Telefonica | Ricardo Serrano |
```

Comment:
```suggestion

```
After internal alignment we have considered that  we cannot stay on as maintainer. In any case we will follow very closely the evolution of the subgroup and count on us as **contributors**. 

Sorry for the missunderstanding